bower.json

```diff
@@ -4,7 +4,8 @@
 	"description": "AMD plugins for RequireJS",
 	"dependencies": {
 		"lie": ">=2.8",
-		"requirejs": "2.1.x"
+		"requirejs": "2.1.x",
+		"text": "requirejs-text#2.0.14"
```

Comment:
Did you test that the mapping of requirejs-text to text does not break anything when used with delite for example ? Also could you change the version used to 2.0.x so we keep consistency with other ibm-js projects.